cmd/tunneld/tunneld.go

```diff
@@ -138,3 +150,87 @@ func fatal(format string, a ...interface{}) {
 	fmt.Fprint(os.Stderr, "\n")
 	os.Exit(1)
 }
+
+func startAutocert(opts *options, server *tunnel.Server, logger log.Logger) {
```

Comment:
Could we move all the Let's Encrypt functions to a separate file `autocert.go` (and tests too).